api/apps/articles/model/save.coffee

```diff
@@ -62,11 +62,13 @@ getDescription = (article) =>
   text
 
 removeStopWords = (title) ->
-  title = title.replace(/[.,\/#!$%\^&\*;:{}=\—_`’~()]/g," ")
-  newTitle = _.difference(title.split(' '), stopWords.stopWords).join(' ')
-  return newTitle
+  title = title.replace(/[0-9.\/#!$%\^\*;{}=_`’~()]/g,"")
```

Comment:
@owendodd also would be good for you to check out the full list of stopwords -- https://github.com/eessex/positron/blob/2a2f784f70540b5b465e45437152d608d6246380/api/lib/stopwords.coffee